source/cloud-native-platform/new-component/helm-chart.html.md.erb

```diff
@@ -169,7 +169,7 @@ Performing a release consists of the following steps:
 
 #### Setting up an AzureDevOps chart build
 
-Join the `dcd_group_ado_user_v2` AzureAD group, this will give you:
+Join the `dcd_group_ado_user_v2` MicrosoftEntraID group, this will give you:
```

Comment:
this is the terminology according to the docs:
https://learn.microsoft.com/en-us/entra/fundamentals/how-to-manage-groups
```suggestion
Join the `dcd_group_ado_user_v2` Microsoft Entra group, this will give you:
```

Reply:
change to microsoft entra.